src/components/TodoList.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC, memo } from 'react';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+interface Props {
+  todos: Todo[],
+  tempTodo: Todo | null,
+  onDelete: (id: number) => void,
+  onChange: (id: number, property: Partial<Todo>) => void
+}
+
+export const TodoList: FC<Props> = memo((props) => {
```

Comment:
Usually we destruct properties right here in function arguments

Reply:
Not agree. destructuring 'props' in a separate line is more comfortably

Reply:
Fixed